### ISSUE TITLE
Background service continues running when "Enable Scheduler" is OFF

### ISSUE THREAD

**Issue by h3298** [robweber/xbmcbackup#138]
Backup still appears in **Settings | System | Add-ons | Running** when scheduled backups are disabled

**Comment by robweber** [comment]
Correct, this is by design. The toggle in the settings turns the scheduler on/off but the service continues to run. This is so that IF the service is ever turned back on it can be detected and the scheduler re-enabled. You can find the relevant information [starting here](https://github.com/robweber/xbmcbackup/blob/master/scheduler.py#L66 in the source code. There is no way to start/stop the service directly from the toggle in the settings. Some kind of monitor must be running to detect when the settings change.
